kivy/uix/textinput.py

```diff
@@ -2585,6 +2587,55 @@ def keyboard_on_textinput(self, window, text):
             self.delete_selection()
         self.insert_text(text, False)
 
+    # current imo composition in progress by the IME system, or '' if nothing
+    _imo_composition = StringProperty("")
```

Comment:
You can remove `""` as empty string is a default value for `StringProperty`.

Reply:
I prefer being explicit over implicit here.